src/containers/ListingPage/ListingPage.jsx

```diff
@@ -166,15 +167,33 @@ const ListingPage = ({ t, locale, location }) => {
     setFilters(mapTagsToFilters(tags));
   };
 
+  const useDebounce = (val, delay) => {
+    const [debouncedVal, setDebouncedVal] = useState(val);
+    useEffect(() => {
+      const handler = setTimeout(() => {
+        setDebouncedVal(val);
+      }, delay);
+      return () => clearTimeout(handler);
+    }, [val, delay]);
+    return debouncedVal;
+  };
+
+  const debouncedSearchVal = useDebounce(searchValue, 200);
+
+  useEffect(() => {
+    if (debouncedSearchVal) getConcepts(page);
+  }, [debouncedSearchVal]);
+
   const getConcepts = async page => {
-    const replace = page === 1;
+    const replace = page === 1 || !debouncedSearchVal.length;
```

Comment:
Kva gjør egentlig denne?

Reply:
Den var tenkt for at vi skal få lastet en clean side med alle relevante resultater dersom man har søkt opp noe og så  fjerner innholdet i søkefeltet. Ser at det ikke fungerer helt som jeg ønska nå, så skal ta en runde til over det.